webapp/cube/views.py

```diff
@@ -314,10 +172,16 @@ def get_microcerts(badgr_issuer, badge_certified, ua_api, badgr_api, edx_api):
 
             courses.append(course)
 
+    edx_register_url = f"{edx_url}{flask.request.base_url}"
+    if flask.request.args.get("test_backend") == "true":
+        edx_register_url = edx_register_url + "?test_backend=true"
+
     return flask.jsonify(
         {
             "account_id": account["id"] if account else None,
             "stripe_publishable_key": get_stripe_publishable_key(),
+            "edx_user": edx_user,
+            "edx_register_url": edx_register_url,
```

Comment:
There is an error with the edx register url. The register redirect is sending the user back to `microcerts.json` instead of /microcers. That is because `flask.request.base_url` is returning now the .json path. 

![image](https://user-images.githubusercontent.com/6387619/148259866-cf589ac9-b4a7-448b-bd3a-9229600fa431.png)